tests/installation/partitioning/lvm_no_separate_home.pm

```diff
@@ -14,10 +14,12 @@
 use parent 'y2_installbase';
 use strict;
 use warnings FATAL => 'all';
+use testapi;
 
 sub run {
-    my $partitioner = $testapi::distri->get_partitioner();
-    $partitioner->edit_proposal(is_lvm => 1, has_separate_home => 0);
+    my $partitioner    = $testapi::distri->get_partitioner();
+    my $multiple_disks = get_var('NUMDISKS', 1) > 1 ? 1 : 0;
+    $partitioner->edit_proposal(is_lvm => 1, has_separate_home => 0, multiple_disks => $multiple_disks);
```

Comment:
You're passing `multiple_disks` method parameter here, but it is not used in `edit_proposal` method. Probably you've forgot to add it in `edit_proposal` or remove here:)

Reply:
I add this changes following https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11822

Reply:
Ah, sorry:)